### PR TITLE
Feature/gh actions

### PR DESCRIPTION
**Pull request by turboladen**

### CODE REVIEW

File: ffi-gdal.gemspec [hunk 1]
```diff
@@ -31,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'fakefs'
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rubocop'
-  spec.add_development_dependency 'rubocop-checkstyle_formatter'
+  spec.add_development_dependency 'rubocop', '>= 1.0'
+  spec.add_development_dependency 'rubocop-performance'
```
Comment (turboladen) [inline ffi-gdal.gemspec:34]: Add this to the config

Comment (turboladen) [reply ffi-gdal.gemspec:34]: done.

File: Rakefile [hunk 2]
```diff
@@ -6,9 +6,6 @@ require 'rubocop/rake_task'
 
 RuboCop::RakeTask.new do |t|
   t.fail_on_error = false
```
Comment (tindron) [inline Rakefile:8]: I think it's safe to remove this, it was only added for teamcity

**Review by tindron — APPROVED** [review]

File: Rakefile (hunk 2 above)
Comment (turboladen) [reply Rakefile]: removed

**Pull request merged**